README.md

```diff
@@ -3,27 +3,39 @@ An app to record what books are in a library and who borrows them. It allows to
 
 ## Built With
 Ruby
+
 ## Getting Started
+
 To get a local copy up and running follow these simple example steps.
 
-## Prerequisites
-Ruby
-## Install
-Go to the terminal and cd into the directory of your preference.
-Run git clone githttps://github.com/basitali111/OOP-school-library.git to clone the app.
-Open the project in a code editor of your choice that is configured to use Ruby.
-Run ruby filename.rb to exectue any of the ruby files
+### Prerequisites
+
+- Ruby
 
+### Install
+
+1. Go to the terminal and `cd` into the directory of your preference.
+2. Run `git clone git@github.com:basitali111/OOP-school-library.git` to clone the app.
+3. Open the project in a code editor of your choice that is configured to use Ruby.
+4. Run `ruby main.rb` to start the app
+
+## Authors
 
 👤 **Basit Ali**
 
 - GitHub: [@githubhandle](https://github.com/basitali111)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/basit-ali-jobs/)
 - Twitter: [Twitter](https://twitter.com/BasitAl35031734)
 
+👤 **Alphayo Wakarindi**
+
+- GitHub: [@githubhandle](https://github.com/alphayowakarindi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/alphayo-wakarindi-15a825236/)
+- Twitter [Twitter](https://twitter.com/alphayowakarind)
+
```

Comment:
- [ ] [OPTIONAL] Your readme is very descriptive and well done :clap: But I would like to recommend you add some command lines to tell how to run the test to someone who doesn't know for example. That will be helpful and make your readme more professional :smile: 

#### Example
 
![Screenshot from 2022-10-28 10-32-15](https://user-images.githubusercontent.com/6615394/198567849-2b3d4ec4-0354-4919-8212-eafb3898fe9d.png)

Reply:
Hi @codecaiine, Thank you so much for taking the time to leave this excellent review. We agree with you, we will add that.